homeassistant/components/conversation/chat_log.py

```diff
@@ -42,6 +42,9 @@ def async_get_chat_log(
         # If a chat log is already active and it's the requested conversation ID,
         # return that. We won't update the last updated time in this case.
         if chat_log.conversation_id == session.conversation_id:
+            if user_input is not None:
```

Comment:
This was a bug. When a chat log was requested while one was active, the user input wouldn't be processed.